### ISSUE TITLE
[Keyboard Navigation - Panel Query Editor]: Cannot access elements which are only visible on hover

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#69388]
**Steps to reproduce**:
Open any dashboard. Create a panel with a DS Prometheus. Add a Sum field. Navigate to the new fields with a keyboard. Notice you cannot select any of the options inside the Sum field because they are only visible on hover

https://github.com/grafana/grafana/assets/13227501/59c690df-4d12-4cd0-87c5-ef3fe7c62929

**Expected Result**
Can navigate to any options in any field using only keyboard.

**Environment**:
- Grafana version: 10.0.0

**Comment by gtk-grafana** [comment]
This appears to be an issue with the icons in the Aggregation component, they are only visible on hover. Although they can still be selected via the keyboard, the outline is not visible, which is still not great for accessibility.

@grafana/observability-metrics will take a look and see if this can be fixed by updating the components in the Prometheus query builder, or if the grafana UI needs to be updated.
